test/html/test_html_table.py

```diff
@@ -84,13 +85,15 @@ def test_html_table_with_img(caplog, tmp_path):
         """<table>
         <tr>
             <td width="50%">
-                <img src="test/image/png_images/affc57dfffa5ec448a0795738d456018.png" height="235" width="435"/>
+                <img src="{}/test/image/png_images/affc57dfffa5ec448a0795738d456018.png" height="235" width="435"/>
             </td>
             <td width="50%">
-                <img src="test/image/image_types/insert_images_insert_png.png" height="162" width="154"/>
+                <img src="{}/test/image/image_types/insert_images_insert_png.png" height="162" width="154"/>
             </td>
         </tr>
-    </table>"""
+    </table>""".replace(
```

Comment:
String interpolation seems more straightforward for this... (and below)

Reply:
Hi @gmischler , agree but that code is not part of this PR.

Reply:
Well, you did add the "replace()" part, so technically... :wink: